import-scripts/clone_db_wrapper.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
I suggest a more specific filename because the scripts directory is such a big mix of scripts for all kinds of purposes. Perhaps : genie-import-airflow-task-clone-db.sh